### PR TITLE
Compilation

### PR DESCRIPTION
**Pull request by beatadelura**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+var gulp = require("gulp"),
+    babelify = require('babelify'),
+    browserify = require("browserify"),
+    connect = require("gulp-connect"),
+    source = require("vinyl-source-stream"),
+    sass = require('gulp-sass')
+;
+
+//Default task. This will be run when no task is passed in arguments to gulp
+gulp.task("default",["copyStaticFiles", "build", "startServer", "watch"]);
```
Comment (MatteoWebDesigner) [inline gulpfile.js]: I am not sure if it is a problem only on my machine but when I run gulp the build fail, looks like startServer has got some problems.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+//Copy static files from html folder to build folder
+gulp.task("copyStaticFiles", function(){
+    return gulp.src("./src/html/*.*")
+    .pipe(gulp.dest("./build"));
+});
+
+//Convert ES6 ode in all js files in src/js folder and copy to 
+//build folder as bundle.js
+
+gulp.task("build", function(){
```
Comment (MatteoWebDesigner) [inline gulpfile.js]: build task should incude the genaration of all static resources, you could call this task js, default task does not include styles task

File: gulpfile.js (hunk 1 above)
Comment (beatadelura) [inline gulpfile.js]: 🤔 It works for me..

File: gulpfile.js (hunk 2 above)
Comment (beatadelura) [inline gulpfile.js]: bitmoji

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+gulp.task('styles', function() {
+    gulp.src('./src/styles/**/*.scss')
+        .pipe(sass().on('error', sass.logError))
+        .pipe(gulp.dest('./build/styles'));
+});
+
+
+//Start a test server with doc root at build folder and 
+//listening to 9001 port. Home page = http://localhost:9001
+gulp.task("startServer", function(){
```
Comment (beatadelura) [inline gulpfile.js]: @MatteoWebDesigner It works now as I expected, so I leave as it is for now.

Many thanks Matteo for help! 😃

**Pull request merged**